src/schemas/User.json

```
@@ -1,18 +1,18 @@
 {
   "title": "User",
   "type": "object",
-  "required": ["name", "last_name"],
+  "required": ["firstName", "lastName"],
```

@Ralexs0096 Could you please reorder the placement of this key to come after the 'properties' key, please. Since the 'properties' key already defines 'firstName' and 'lastName', it's more logical to place this key afterwards. While JavaScript hoisting might not significantly impact performance in this case, adhering to best practices is advisable. Thanks!

```suggestion
  "required": ["firstName", "lastName"],
```

---

in the JSON schema convention, it is not relevant to put the "required" field before or after, but when reading it it is easier to have first the context of the properties and which ones are required, so I will move it, and we will add it as a convention in our schemas. Thanks for the contribution @AlvisonHunterArnuero .